src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -77,6 +78,26 @@ function WorkspaceTaxesPage({policy}: WorkspaceTaxesPageProps) {
         </View>
     );
 
+    const headerButtons = (
+        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
+            <Button
+                medium
+                success
+                onPress={() => {}}
+                icon={Expensicons.Plus}
+                text="Add rate"
```

Comment:
`translate(...)`

Reply:
Waiting for translators confirmation [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1709888428303999?thread_ts=1709801973.344319&cid=C01GTK53T8Q)